mmv1/templates/terraform/examples/backend_service_cache_include_named_cookies.erb

```diff
@@ -0,0 +1,11 @@
+resource "google_compute_backend_service" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
I think this file's name needs to be `backend_service_include_named_cookies.tf.erb`! Sorry for the confusion there.

Reply:
Oh, sorry, no, it was the file extension I was referring to - sorry! `backend_service_cache_include_named_cookies.tf.erb` is what we need this file called.